.github/workflows/release.yml

```diff
@@ -52,46 +52,14 @@ jobs:
         run: |
           export PATH=${PATH}:`go env GOPATH`/bin
           go get -v -t -d ./...
-      - name: Build govvv
+      - name: Build release artifacts
         run: |
-          go build -modfile .bingo/govvv.mod -o=. github.com/ahmetb/govvv
-      - name: Build gox
-        run: |
-          go build -modfile .bingo/gox.mod -o=. github.com/mitchellh/gox
-      - name: Compile
-        run: |
-          VERSION=${GITHUB_REF##*/}
-          ./gox -osarch="linux/amd64 linux/386 linux/arm darwin/amd64 windows/amd64" -output="pow-{{.OS}}-{{.Arch}}" -ldflags="$(./govvv -flags -version $(VERSION) -pkg $(go list ./buildinfo))" ./cmd/pow
-      - name: Collect artifacts
-        run: |
-          VERSION=${GITHUB_REF##*/}
-          OUT=release/cli
-          mkdir -p ${OUT}
-          mkdir -p tmp
-          cp LICENSE tmp/
-          cp README.md tmp/
-          cp dist/install tmp/
-          cd tmp
-          declare -a arr=("darwin-amd64" "linux-amd64" "linux-386" "linux-arm" "windows-amd64.exe")
-          for i in "${arr[@]}"
-          do
-              OSARCH=${i%.*}
-              EXT=$([[ "$i" = *.* ]] && echo ".${i##*.}" || echo '')
-              cp ../pow-${i} pow${EXT}
-              if [ "${EXT}" == ".exe" ]; then
-                  zip pow_${VERSION}_${OSARCH}.zip LICENSE README.md pow${EXT}
-                  mv pow_${VERSION}_${OSARCH}.zip ../${OUT}/
-              else
-                  tar -czvf pow_${VERSION}_${OSARCH}.tar.gz LICENSE README.md install pow
-                  mv pow_${VERSION}_${OSARCH}.tar.gz ../${OUT}/
-              fi
-          done
-          cd .. && rm -rf tmp
-          echo $(ls ./release/cli)
```

Comment:
All this logic is now in the `Makefile` and is simplified there.

Reply:
Nice... easier to test changes locally than artificially running releases.